include/opendht/dht_interface.h

```diff
@@ -214,11 +214,11 @@ class OPENDHT_PUBLIC DhtInterface {
 
     virtual std::vector<SockAddr> getPublicAddress(sa_family_t family = 0) = 0;
 
-    Logger DHT_LOG;
+    static Logger DHT_LOG;
```

Comment:
the logger should be independent per node. For instance each node might log to a different file.

Reply:
Oh. Indeed...
I think you already told me that. I'll do it asap. Just need to provide docs first. If I will not do it today, I'll do it next week should be easy to fix :)